### DISCUSSION TITLE
Questions about Proxy

### DISCUSSION THREAD

**Discussion by camm9909** [slskd/slskd#540] [Q&A] [answered]
I love this! I used to run SoulseekQT with Wine & Xvfb on a headless server. I'm now using this and it's great to have remote web access and pushbullet notifications. I love being along for the ride. Thanks!

I haven't been able to get the proxy to work. I have local storage and a remote VPS (ubuntu), I'm currently running the client-server app on the VPS (so as to hide my home IP) and using SSH filesystem tunnel to my home storage (I know). I'm wondering what's a good way of doing a reverse proxy to work, if possible?

Thanks again :)

**Comment by jpdillingham** [comment, marked as answer]
Glad you're enjoying it!

The app is built to natively support operation behind a reverse proxy using the `URL_BASE` setting described [here](https://github.com/slskd/slskd/blob/master/docs/config.md#basic).  The rest of the configuration will depend on what you're using.  I recently added some documentation for [nginx](https://github.com/slskd/slskd/blob/master/docs/reverse_proxy.md); you should be able to translate this to other webservers.
